etcdmgmt/utils.go

```diff
@@ -0,0 +1,115 @@
+package etcdmgmt
+
+import (
+	"io/ioutil"
+	"net/url"
+	"path"
+
+	"github.com/gluster/glusterd2/gdctx"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/coreos/etcd/embed"
+	"github.com/ghodss/yaml"
+	config "github.com/spf13/viper"
+)
+
+// EtcdConfigFile is path to etcd config file
+var EtcdConfigFile string
+
+type etcdMinimalConfig struct {
```

Comment:
Also the method `readEtcdConfig` which deals with unexported structure `etcdMinimalConfig` is also no longer public function.